### PR TITLE
Fix the inconsistency issue of LZ4 performance across different hardware during autotest.

### PR DESCRIPTION
**Pull request by Hparty**
Fix the inconsistency issue of LZ4 performance across different hardware during autotest.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -85,12 +87,14 @@ endif ()
 if (PAG_BUILD_TESTS)
     set(PAG_USE_FREETYPE ON)
     set(PAG_USE_HARFBUZZ ON)
+    set(PAG_USE_SOFTWARE_LZ4 ON)
```
Comment (domchen) [inline CMakeLists.txt:90]: 改个名字，PAG_USE_SYSTEM_LZ4, 然后默认值ON，遇到Tests的时候改成OFF

Comment (Hparty) [reply CMakeLists.txt:90]: ok

File: CMakeLists.txt [hunk 2]
```diff
@@ -53,8 +53,10 @@ endif ()
 
 if (MACOS OR IOS)
     option(PAG_BUILD_FRAMEWORK "Allow build framework" ON)
+    set(PAG_USE_SYSTEM_LZ4 ON)
```
Comment (domchen) [inline CMakeLists.txt:56]: 你要加的是Option默认值，不能是Set，否则外部传入的就被你覆盖了。

Comment (Hparty) [reply CMakeLists.txt:56]: 👌

**Review by domchen — APPROVED** [review]

**Pull request merged**